explorer/deploy/webAppCreator.ts

```diff
@@ -447,6 +468,12 @@ class WebsiteStep extends WebAppCreatorStepBase {
                 await vscode.window.showWarningMessage(nameAvailability.message);
             }
         }
+        try {
+            await this.loginCredentials();
```

Comment:
>loginCredentials [](start = 23, length = 16)

Can you rename to something like acquireRegistryLoginCredentials? It's not clear from the name what credentials are about since it's in WebsiteStep class.

Reply:
That seems reasonable to me